packages/react-core/src/components/Spinner/Spinner.tsx

```diff
@@ -33,36 +31,27 @@ export const Spinner: React.FunctionComponent<SpinnerProps> = ({
   className = '',
   size = 'xl',
   'aria-valuetext': ariaValueText = 'Loading...',
-  isSVG = true,
   diameter,
   isInline = false,
   'aria-label': ariaLabel,
   'aria-labelledBy': ariaLabelledBy,
   ...props
 }: SpinnerProps) => {
-  const Component = isSVG ? 'svg' : ('span' as any);
+  const Component = 'svg' as any;
```

Comment:
```suggestion
  const Component = 'svg';
```
We might not need to cast this if it isn't causing type errors... WDYT @gitdallas ?

Reply:
It's used many places similarly:
<img width="427" alt="image" src="https://user-images.githubusercontent.com/5322142/215205564-b809d709-71d3-44a4-82a1-dd3afd434f60.png">

without the casting we get this:
<img width="785" alt="image" src="https://user-images.githubusercontent.com/5322142/215205666-e194b66f-0e72-4846-a3f1-1b6bd78f7aa7.png">

not sure if there's a better solution

Reply:
Ah makes sense, thank you for clarifying!

Reply:
I think I should be able to remove casting if i have the proper prop type.. trying `export interface SpinnerProps extends React.SVGProps<SVGSVGElement>` and seeing an issue with the `style.--pf-c-spinner--diameter` prop... which honestly doesn't look right anyway.. so i'm looking into that

Reply:
alright @jenny-s51 - i was able to remove the component cast by updating the props type, just had an issue with svg style prop not having '--pf-c-spinner--diameter`
![image](https://user-images.githubusercontent.com/5322142/215221870-1e0e942f-6a67-4bc2-b776-ff1fc54ff44c.png)

which i worked around by casting that. curious if @nicolethoen has any thoughts about this

Reply:
I think you can cast the style object to `CSSProperties` rather than casting the variable name as any.
And do can you avoid setting the 'Component' var and just use `svg` as the element on line 43?

Reply:
good point, thanks for taking a step back 😄